<launch>

    <group ns="shadow">
        <include file="$(find ros_aws_iot_bridge)/launch/shadow.launch">
        
            <arg name="topicname/state" value="/state"/>

            <arg name="topicname/state/delta" value="state/delta"/>
            <arg name="topicname/state/report" value="state/report"/>
            <arg name="topicname/state/report_and_desire" value="state/report_and_desire"/>

        </include>
    </group>

</launch>